tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala

```diff
@@ -163,7 +165,9 @@ class HoverTermSuite extends BaseHoverSuite {
     """|def this(name: String, age: Int): Foo
        |""".stripMargin.hover,
     compat = Map(
-      "3" -> "class Foo: Foo".hover
+      "3" ->
```

Comment:
you can remove this compat case

Reply:
👍 https://github.com/scalameta/metals/pull/3854/commits/e1956bf9efa8cdc0cf802c10ce92a14d313e7b47